kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx

```diff
@@ -97,13 +97,19 @@ const Details: React.FC<Props> = ({
               </tr>
             </thead>
             <tbody>
-              {items.map((consumer) => (
-                <ListItem
-                  key={consumer.consumerId}
-                  clusterName={clusterName}
-                  consumer={consumer}
-                />
-              ))}
+              {items.length > 0 ? (
+                items.map((consumer) => (
+                  <ListItem
+                    key={consumer.consumerId}
+                    clusterName={clusterName}
+                    consumer={consumer}
+                  />
+                ))
+              ) : (
+                <tr>
+                  <td colSpan={10}>No active consumer group</td>
+                </tr>
+              )}
```

Comment:
there is no need to check length for map.
```
<tbody>
  {items.map((consumer) => (
                  <ListItem
                    key={consumer.consumerId}
                    clusterName={clusterName}
                    consumer={consumer}
                  />
                ))}
   {{items.length === 0 && (
     <tr>
                  <td colSpan={10}>No active consumer group</td>
                </tr>
  )}

```

Reply:
done

Reply:
I see it is not done